app/assets/stylesheets/active_admin/_base.scss

```diff
@@ -33,12 +33,14 @@
 @import "active_admin/structure/main_structure";
 @import "active_admin/structure/title_bar";
 
+* {
+  box-sizing: border-box;
+}
+
 body {
-  @include sans-family;
-  line-height: 150%;
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Comment:
Properties should be ordered background, color, font-family, font-size, line-height